srcpkgs/qt5-webkit/template

```diff
@@ -36,7 +36,7 @@ case "$XBPS_TARGET_MACHINE" in
 		;;
 	i686*) 	# try to reduce memory footprint when linking by using gold
 		nodebug=1
-		configure_args+=" -DENABLE_ALLINONE_BUILD=OFF"
+		configure_args+=" -DUSE_LD_GOLD=ON"
```

Comment:
is this on purpose?

Reply:
I didn't clean up my branch before pushing. Will clean it once the local build finished.

Reply:
:+1: